modules/apps/transfer/keeper/relay.go

```diff
@@ -62,6 +64,10 @@ func (k Keeper) SendTransfer(
 		return types.ErrSendDisabled
```

Comment:
This is a nit and no need to block the PR on this, but now that you added tests for the msg server, maybe we can also had a test case for the situation where send is disabled.

Reply:
looks like this test case is missing for receive as well. Would prefer it be done in a separate pr